### PR TITLE
feat: [modal migration] suggest new source

### PR DESCRIPTION
**Pull request by el**
## Changes
Screenshot 2022-11-18 at 16 43 45
Screenshot 2022-11-18 at 16 43 54

WT-753 #done

### CODE REVIEW

File: packages/shared/src/components/Loader.tsx [hunk 1]
```diff
@@ -12,7 +12,15 @@ export function Loader({
...
 }: LoaderProps): ReactElement {
   return (
-    <div {...props} className={classNames('w-5 h-5', className, styles.loader)}>
+    <div
+      {...props}
+      className={classNames(
+        'w-5 h-5',
+        className,
+        styles.loader,
+        invertColor && 'invert',
```
Comment (el) [inline packages/shared/src/components/Loader.tsx:21]: invert wasnt working, so I fixed that

File: packages/shared/src/components/modals/SubmitArticleModal.tsx [hunk 2]
```diff
@@ -182,7 +182,7 @@ export default function SubmitArticleModal({
   return (
     <Modal
       {...modalProps}
-      kind={Modal.Kind.FlexibleCenter}
```
Comment (rebelchris) [inline packages/shared/src/components/modals/SubmitArticleModal.tsx:185]: Why did this one change?

**Review by rebelchris — CHANGES_REQUESTED** [review]
One remark for the other modal.

And for this one, it looks a bit weird with this placeholder.
Screenshot 2022-11-18 at 16 09 23

Maybe double check with design what they would like this to do?

File: packages/shared/src/components/modals/SubmitArticleModal.tsx (hunk 2 above)
Comment (rebelchris) [reply packages/shared/src/components/modals/SubmitArticleModal.tsx:185]: Pretty sure they should both be FlexibleCenter right?

Comment (el) [reply packages/shared/src/components/modals/SubmitArticleModal.tsx:185]: Their height changes based on the content so it causes them to shift, that's why I changed them both. in the design one of they were stated as flexible top:

https://www.sketch.com/s/2867ec63-0966-4647-b1d7-b6fe696ba251/a/dlpl9AZ

**Review by rebelchris — APPROVED** [review]
Looks solid now and double checked with design on the flexibleTop which is right

**Pull request merged**